### DISCUSSION TITLE
markerclusterer as Component

### DISCUSSION THREAD

**Discussion by luiscrjunior** [visgl/react-google-maps#364] [Ideas]
Hello!

Is there a plan to wrap `markerclusterer` logic into a new component (`<MarkerClusterer />` maybe)?

Don't know if it was considered/discussed, or if there's some technical limitation, or, yet, if it's out-of-scope of this lib.

If possible, I can work into a PR (with tests, docs).

**Comment by usefulthink** [comment]
There are no plans for that yet, but I am aware that marker clustering seems to be one of the features that a lot of people are using (at least judging by the discussions here). So definitely open to improving the support for that. 

Can you roughly sketch what you would like such a component to look like from the users perspective? That would be a good start for a discussion. Also, if you want to just go ahead and write some code to see what works, make sure to read this first: https://visgl.github.io/react-google-maps/docs/contributing

**Comment by luiscrjunior** [comment]
Nice, wanna think about and bring some proposal.
